docs/api-network.rst

```diff
@@ -297,7 +297,7 @@ Account Methods
         >>> accounts[0].balance() == "100 ether"
         True
 
-.. py:classmethod:: Account.deploy(contract, *args, amount=None, gas_limit=None, gas_price=None, nonce=None, required_confs=1)
+.. py:classmethod:: Account.deploy(contract, *args, amount=None, gas_limit=None, gas_price=None, nonce=None, required_confs=1, allow_revert=False, silent=False, publish_source=False,)
```

Comment:
I wonder if `silent` should be `True ` by default. I would like to avoid the print of `Verification complete. Result: Pass - Verified` for end users.

Reply:
I think some feedback is nice, especially for longer calls and for newer users. Advanced users will know how to disable it. 
@iamdefinitelyahuman you have an opinion on this?

Reply:
The `silent` kwarg is applying to both verification and deployment..  I don't think we'd want to silence the output on the actual deployment action, so unless we split it into 2 silent kwargs, I vote it's `False` by default.

Reply:
I think two args is overkill for this. Since verification takes around 30 secs+, I think it is nice to let the user know why it is stalling.